### DISCUSSION TITLE
Remove 0% badge on checkbox questions for "poll" questions

### DISCUSSION THREAD

**Discussion by firasm** [PrairieLearn/PrairieLearn#6303] [Q&A] [answered]
Hello 

I've got a poll-type question setup on PrairieLearn, and I've figured out how to give everyone 100% just for answering the question (`data['score'] = 1` in the `grade()` function of server.py), but I can't quite figure out how to remove that 0% badge. Is there currently a way to do that through some combination of the available settings?

<img width="951" alt="Screen Shot 2022-09-09 at 10 30 55 AM" src="https://user-images.githubusercontent.com/2507459/189410631-b2aaadcf-834c-4812-84a0-6baae4262176.png">

[Here are the docs](https://prairielearn.readthedocs.io/en/latest/elements/#pl-checkbox-element)

- I tried setting them all to be correct and then `maxSelect = 1` but PL is too smart for that hack (threw an error).
- I also tried multiple-choice element but that's even less flexible than the checkbox

**Comment by jonatanschroeder** [comment, marked as answer]
You can set `data["partial_scores"][field_name]` to `{ 'score': 1 }` in your `grade()` function.

**Reply by firasm** [reply]
Thanks! that did it!

Through this trick I can also completely remove that badge!
